D1371R0.pandoc

```diff
@@ -337,7 +337,25 @@ inspect (v) {
 }
 ```
 
-[ _Note:_ Even though `_` is a valid identifier, it does not introduce a name. --- _end note_ ]
+[ _Note:_ Even though `_` is a valid identifier, it does not introduce a name
+as doing so would require all values matched and represented by the wildcard
```

Comment:
It's more-so that repeated identifiers, if they introduce a name, are redeclaration errors. e.g., `[x, x]` is a redeclaration error, it doesn't match pairs that contain the same value.